game_app/common/buttons.py

```diff
@@ -101,19 +102,20 @@ def set_disabled(self, new_disabled):
 
     def on_pressed(self):
         if self._disabled:
-            self._disabled_click_sound.play()
+            if self._sounds_on:
+                self._disabled_click_sound.play()
```

Comment:
maybe:

if self._disabled and self._sounds_on:
                self._disabled_click_sound.play()

Reply:
no wlasnie nie, bo wtedy przy wylaczonym dzwieku by weszlo w else i mimo wylaczenia przycisku przycisk by zadzialal